### PR TITLE
18205 Fixed incorrect entity types

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#18205

*Description of changes:*
- app version = 5.2.11
- changed SP -> FR in Business Lookup list
- changed ULC-> UL in Business Lookup list

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://namerequest-dev--pr-730-cvs5k9nw.web.app

File: src/list-data/request-action-mapping.ts [hunk 1]
```diff
@@ -100,22 +100,22 @@ export const BusinessLookupRequestActions = [
...
+  EntityTypes.BC, // Benefit Company
+  EntityTypes.BEN, // invalid?
   EntityTypes.C,
   EntityTypes.CC,
   EntityTypes.CCC,
   EntityTypes.CUL,
   EntityTypes.CP,
-  EntityTypes.CR,
+  EntityTypes.CR, // BC Limited Company
+  EntityTypes.FR, // Sole Proprietorship
```
Comment (severinbeauvais) [inline src/list-data/request-action-mapping.ts:111]: I tested FR by creating a name change NR with FM1056109 and verifying that corpnum was populated.

File: src/list-data/request-action-mapping.ts [hunk 2]
```diff
@@ -100,22 +100,22 @@ export const BusinessLookupRequestActions = [
...
   EntityTypes.GP,
   EntityTypes.LLC,
   EntityTypes.LP,
   EntityTypes.PA,
   EntityTypes.PAR,
   EntityTypes.SO,
-  EntityTypes.SP,
-  EntityTypes.ULC,
+  EntityTypes.UL,
```
Comment (severinbeauvais) [inline src/list-data/request-action-mapping.ts:118]: I tested UL by creating a name change NR for BC0871495 and verifying that corpnum was populated.

**Review by JazzarKarim — APPROVED** [review]

File: src/list-data/request-action-mapping.ts [hunk 3]
```diff
@@ -100,22 +100,22 @@ export const BusinessLookupRequestActions = [
 /** Entity types that require business lookup. */
 export const BusinessLookupEntityTypes = [
   EntityTypes.A,
-  EntityTypes.BC,
-  EntityTypes.BEN,
+  EntityTypes.BC, // Benefit Company
+  EntityTypes.BEN, // invalid?
```
Comment (severinbeauvais) [inline src/list-data/request-action-mapping.ts:104]: I think this is invalid but I'm not about to remove it in a hotfix without fully understanding how this list is used 😛 

I also checked the other entries in this list, and I think they're OK, but there may be some types missing.

Comment (leodube-aot) [reply src/list-data/request-action-mapping.ts:104]: Yeah I agree I had a brief look through namex and I don't believe BEN is valid, but it is probably not worth removing it on a hotfix.

**Review by leodube-aot — APPROVED** [review]

**Pull request merged**